### PR TITLE
{chem}[intel/2015a] LAMMPS (WIP)

### PR DESCRIPTION
**Pull request by boegel**
cc @ocaisa 

requires https://github.com/hpcugent/easybuild-easyblocks/pull/594

### CODE REVIEW

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-20150210-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# see http://lammps.sandia.gov/bug.html for overview of stable versions and matching date/SVN rev/Git tag
+version = '20150210'
+
+homepage = ''
+description = """LAMMPS is a classical molecular dynamics code, and an acronym for Large-scale Atomic/Molecular
+ Massively Parallel Simulator. LAMMPS has potentials for solid-state materials (metals, semiconductors) and
+ soft matter (biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be used to model atoms or,
+ more generically, as a parallel particle simulator at the atomic, meso, or continuum scale."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.14-no-OFED'}
```
Comment (boegel) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-20150210-intel-2015a.eb]: note: this is pretty wrong...

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-20150210-intel-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+toolchain = {'name': 'intel', 'version': '2015a'}
+ 
+source_urls = ['http://lammps.sandia.gov/tars/']
+sources = ['lammps-10Feb15.tar.gz']
+ 
+packages = ['asphere', 'body', 'reax', 'poems']
+ 
+checksums = ['25a432713a3ad4969fdaaa2551d517ef']
+ 
+patches = ['meam.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-20150210-intel-2015a.eb]: @ocaisa: can you provide me this patch file before I pick up on this again?

Comment (ocaisa) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-20150210-intel-2015a.eb]: Done

**Comment by ocaisa** [comment]
Tests fail because easyblock is not yet merged

**Comment by akesandgren** [comment]
@boegel, this PR is being closed for the following reason(s): obsoleted by more recent PRs.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**